php/example_usage.php

```diff
@@ -0,0 +1,19 @@
+<?php require('PaypalIPN.php');
+
+use PaypalIPN;
+
+$ipn = new PayPalIPN();
+
+// Use the sandbox endpoint during testing.
+$ipn->useSandbox();
+$verified = $ipn->verifyIPN();
+if ($verified) {
+    /*
+     * Process IPN
+     * A list of variables is available here:
+     * https://developer.paypal.com/webapps/developer/docs/classic/ipn/integration-guide/IPNandPDTVariables/
+     */
+}
+
+// Reply with an empty 200 response to indicate to paypal the IPN was received correctly.
+header("HTTP/1.1 200 OK");
```

Comment:
You are missing a newline at the end of this file. Note the `red` sign in github.